addons/website_event/data/event_demo.xml

```diff
@@ -51,6 +51,8 @@
         <field name="website_published">True</field>
         <field name="subtitle">Enhance your architectural business and improve professional skills.</field>
         <field name="twitter_hashtag">odoo</field>
+        <field eval="(DateTime.now() - timedelta(days=5)).strftime('%Y-%m-%d 08:00:00')" name="start_sale_date"/>
```

Comment:
When doing an xml record, try to put the name before the val, otherwise a narwhal die everytime you change the order. Poor narwhal.